pkg/process/procutil/process_linux.go

```diff
@@ -244,26 +318,61 @@ func (p *Probe) parseStatusKV(key, value string, sInfo *statusInfo) {
 			sInfo.ctxSwitches.Involuntary = v
 		}
 	case "VmRSS":
-		value := strings.Trim(value, " kB") // remove last "kB"
+		value := strings.Trim(value, " kB") // trim spaces and suffix "kB"
 		v, err := strconv.ParseUint(value, 10, 64)
 		if err == nil {
 			sInfo.memInfo.RSS = v * 1024
 		}
 	case "VmSize":
-		value := strings.Trim(value, " kB") // remove last "kB"
+		value := strings.Trim(value, " kB") // trim spaces and suffix "kB"
 		v, err := strconv.ParseUint(value, 10, 64)
 		if err == nil {
 			sInfo.memInfo.VMS = v * 1024
 		}
 	case "VmSwap":
-		value := strings.Trim(value, " kB") // remove last "kB"
+		value := strings.Trim(value, " kB") // trim spaces and suffix "kB"
 		v, err := strconv.ParseUint(value, 10, 64)
 		if err == nil {
 			sInfo.memInfo.Swap = v * 1024
 		}
 	}
 }
 
+// ensurePathReadable ensures that the current user is able to read the path before opening it.
+// On some systems, attempting to open a file that the user does not have permission is problematic for
+// customer security auditing. What we do here is:
+// 1. If the agent is running as root (real or via sudo), allow the request
+// 2. If the file is a not a symlink and has the other-readable permission bit set, allow the request
```

Comment:
Should it be `If the file is not a symlink`?

Reply:
Sorry I thought that's what the comment was saying.

Reply:
There was an additional `a` in the comment but this was definitely a nit :)